autogpt/llm/base.py

```diff
@@ -140,8 +151,11 @@ class LLMResponse:
     """Standard response struct for a response from an LLM model."""
 
     model_info: ModelInfo
-    prompt_tokens_used: int = 0
-    completion_tokens_used: int = 0
+
```

Comment:
These are not OpenAI specific, I would thing.  Every language model is going to consume tokens in its usage.  That being said, I don't think having this baseclass is helpful.  It's not used anywhere.  I would suggest ditching it and putting in 

```python
@dataclass
class EmbeddingModelResponse:
    """Standard response struct for a response from an embedding model."""

    embedding: List[float] = field(default_factory=list)
    prompt_tokens_used: int

    @property
    def completion_tokens_used(self) -> int:
        return 0

@dataclass
class ChatModelResponse:
    """Standard response struct for a response from a chat LLM."""
    content: Optional[str]
    function_call: Optional[OpenAIFunctionCall]
    prompt_tokens_used: int
    completion_tokens_used: int
```

We do need the `prompt_tokens_used` and `completion_tokens_used` fields as they are used to calculate API costs.

Reply:
I guess they are not, now looking at your metering decorator.  So can drop them.